src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandler.java

```diff
@@ -141,4 +147,19 @@ private static HearingChannel getNextHearingChannel(SscsCaseData caseData) {
                         hearingChannel.getValueTribunals()))
                 .findFirst().orElse(HearingChannel.PAPER);
     }
+
+    private static void updateExcludedPanelMembers(SscsCaseData caseData) {
+        Adjournment adjournment = caseData.getAdjournment();
+        AdjournCasePanelMembersExcluded panelMemberExcluded = adjournment.getPanelMembersExcluded();
+        if (nonNull(panelMemberExcluded)) {
+            PanelMemberExclusions panelMemberExclusions = caseData.getSchedulingAndListingFields()
+                .getPanelMemberExclusions();
+
+            if (panelMemberExcluded.equals(AdjournCasePanelMembersExcluded.YES)) {
+                excludePanelMembers(panelMemberExclusions, adjournment.getPanelMembers());
```

Comment:
I think it would be better to have this qualified rather than static imported. Lucas advised me to only static import very commonly used and easily identifiable methods. This is both niche and used only once in this file.
```suggestion
                SscsUtil.excludePanelMembers(panelMemberExclusions, adjournment.getPanelMembers());
```

Reply:
done